test/js/acme.js

```diff
@@ -0,0 +1,89 @@
+var colors = require("colors");
+var crypto = require("./crypto-util");
```

Comment:
not sure if you care, but `crypto` shadows the name of a standard library module, might be confusing to someone who reads halfway through. https://nodejs.org/api/crypto.html